lib/node_modules/@stdlib/stats/base/ndarray/dcovarmtk/README.md

```diff
@@ -0,0 +1,216 @@
+
+```math
+\mu_x = \frac{1}{N} \sum_{i=0}^{N-1} x_i
+```
+
+<!-- </equation> -->
+
+and
+
+<!-- <equation class="equation" label="eq:population_mean_for_y" align="center" raw="\mu_y = \frac{1}{N} \sum_{i=0}^{N-1} y_i" alt="Equation for the population mean for second array."> -->
+
+```math
+\mu_y = \frac{1}{N} \sum_{i=0}^{N-1} y_i
+```
+
+<!-- </equation> -->
+
+Often in the analysis of data, the true population [covariance][covariance] is not known _a priori_ and must be estimated from samples drawn from population distributions. If one attempts to use the formula for the population [covariance][covariance], the result is biased and yields a **biased sample covariance**. To compute an **unbiased sample covariance** for samples of size `n`,
+
+<!-- <equation class="equation" label="eq:unbiased_sample_covariance" align="center" raw="\operatorname{\mathrm{cov_n}} = \frac{1}{n-1} \sum_{i=0}^{n-1} (x_i - \bar{x}_n)(y_i - \bar{y}_n)" alt="Equation for computing an unbiased sample variance."> -->
+
+```math
+\mathop{\mathrm{cov_n}} = \frac{1}{n-1} \sum_{i=0}^{n-1} (x_i - \bar{x}_n)(y_i - \bar{y}_n)
+```
+
+<!-- </equation> -->
+
+where sample means are given by
+
+<!-- <equation class="equation" label="eq:sample_mean_for_x" align="center" raw="\bar{x} = \frac{1}{n} \sum_{i=0}^{n-1} x_i" alt="Equation for the sample mean for first array."> -->
+
+```math
+\bar{x} = \frac{1}{n} \sum_{i=0}^{n-1} x_i
+```
+
+<!-- </equation> -->
+
+and
+
+<!-- <equation class="equation" label="eq:sample_mean_for_y" align="center" raw="\bar{y} = \frac{1}{n} \sum_{i=0}^{n-1} y_i" alt="Equation for the sample mean for second array."> -->
+
+```math
+\bar{y} = \frac{1}{n} \sum_{i=0}^{n-1} y_i
+```
+
+<!-- </equation> -->
+
+The use of the term `n-1` is commonly referred to as Bessel's correction. Depending on the characteristics of the population distributions, other correction factors (e.g., `n-1.5`, `n+1`, etc) can yield better estimators.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var dcovarmtk = require( '@stdlib/stats/base/ndarray/dcovarmtk' );
+```
+
+#### dcovarmtk( arrays )
+
+Computes the covariance of two one-dimensional double-precision floating-point ndarrays provided known means and using a one-pass textbook algorithm.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+
+var opts = {
+    'dtype': 'float64'
+};
+
+var xbuf = new Float64Array( [ 1.0, -2.0, 2.0 ] );
+var x = new ndarray( opts.dtype, xbuf, [ 3 ], [ 1 ], 0, 'row-major' );
+
+var ybuf = new Float64Array( [ 2.0, -2.0, 1.0 ] );
+var y = new ndarray( opts.dtype, ybuf, [ 3 ], [ 1 ], 0, 'row-major' );
+
+var meanx = scalar2ndarray( 1.0/3.0, opts );
+var meany = scalar2ndarray( 1.0/3.0, opts );
+var correction = scalar2ndarray( 1.0, opts );
+
+var v = dcovarmtk( [ x, y, meanx, meany, correction ] );
```

Comment:
@gururaj1512 Would you mind reordering the ndarrays here to

```
[ x, y, correction, meanx, meany ]
```

?

This ensures that the relative order of the ndarrays matches the order of the arguments in the corresponding strided array function

```
dcovarmtk( N, correction, meanx, x, strideX, meany, y, strideY )
```

Notice: `correction`, `meanx`, and `meany` order.

That applies throughout this PR.